app/javascript/packs/clipboardCopy.js

```diff
@@ -1,5 +1,5 @@
-document.addEventListener('DOMContentLoaded', () => {
-  window.WebComponents.waitFor(() => {
-    import('@github/clipboard-copy-element');
-  });
+window.clipboard = require('clipboard-polyfill');
```

Comment:
why not the `import` syntax here?

Reply:
Import would work here for sure.

```suggestion
import clipboard from 'clipboard-polyfill';

window.clipboard = clipboard;
```

Also, if this is only for Android, consider only adding this polyfill when on Android so that non-Android users don't need to pay the polyfill tax.

Reply:
Thanks for the suggestion! I went ahead and applied it.

Conditionally load in polyfill is a great idea but I would have to remove this package from the app and have it load in via CDN when necessary right?